test/NinjaTools.FluentMockServer.API.Tests/Configuration/ConfigurationServiceTests.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections.Generic;
+using Microsoft.Extensions.Options;
+using Moq;
+using NinjaTools.FluentMockServer.API.Configuration;
+using NinjaTools.FluentMockServer.API.Models;
+using NinjaTools.FluentMockServer.API.Services;
+using NinjaTools.FluentMockServer.Tests.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NinjaTools.FluentMockServer.API.Tests.Configuration
+{
+    public class ConfigurationServiceTests : XUnitTestBase<ConfigurationServiceTests>
+    {
+        /// <inheritdoc />
+        public ConfigurationServiceTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        private IConfigurationService CreateSubject(out Mock<IConfigFileProvider> provider, out Mock<ISetupService> setupService)
+        {
+            var options = Options.Create(new MockServerOptions
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'options'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4765163)